### PR TITLE
Add some tableStyle variables for displaying charts

### PR DESCRIPTION
**Pull request by rdengate**
Per column colour and yAxis min/max values to allow more control over chart display.

### CODE REVIEW

File: lib/Models/TableColumnStyle.js [hunk 1]
```diff
@@ -122,6 +137,12 @@ var TableColumnStyle = function(options) {
     this.featureInfoFields = options.featureInfoFields;
 
     /**
+     * Color for column (css string)
+     * @type {String}
+     */
+    this.color = options.color;
```
Comment (kring) [inline lib/Models/TableColumnStyle.js]: This only applies to the chart, not to the map, right?  A more descriptive name (`chartLineColor`?) would make it clearer, or at least mention in the doc that it only applies to charting.

**Comment by kring** [comment]
Please update CHANGES.md, too.

**Comment by kring** [comment]
Looks good other than those two small things.

**Comment by rdengate** [comment]
Thanks @kring. I have made all changes you suggested.

**Pull request merged**